### DISCUSSION TITLE
Introduce action to mark issues as stale

### DISCUSSION THREAD

**Discussion by kengu** [fleaflet/flutter_map#848] [Ideas]
@johnpryan / @maRci002 / others: We have many issues with follow-up questions that are not answered by OP. Any thoughts on introduction action [close-stale-issues](https://github.com/marketplace/actions/close-stale-issues) to help us mark these as stale?

**Comment by johnpryan** [comment]
Sounds good to me - maybe 60 days is a good place to start?

**Reply by kengu** [reply]
Yes, I think that is a reasonable gracetime

**Reply by kengu** [reply]
An I think we should disable closing stale PRs with `days-before-pr-close: -1`

**Reply by kengu** [reply]
I don't have permissions to add this action though.

**Reply by johnpryan** [reply]
I see- yeah it would be nice to have a `stale` label automatically applied. I'll see if I can add this.
